spec/spec_helper.cr

```diff
@@ -127,7 +123,11 @@ def warnings_result(code, inject_primitives = true)
   compiler.prelude = "empty" # avoid issues in the current std lib
   compiler.color = false
   apply_program_flags(compiler.flags)
-  result = compiler.compile Compiler::Source.new("code.cr", code), output_filename
+  source = [Compiler::Source.new("code.cr", code)]
+  if inject_primitives
+    source.unshift Compiler::Source.new("inject_primitives", %(require "primitives"))
```

Comment:
I think that actually since we are using the compiler with prelude empty, the primitives should already be included, meaning that the `inject_primitives` in this method is useless 🤔

Reply:
Indeed, it doesn't make a difference. I'll remove it.